x/observer/keeper/observer_mapper.go

```diff
@@ -93,24 +93,32 @@ func (k msgServer) AddObserver(goCtx context.Context, msg *types.MsgAddObserver)
 	if msg.Creator != k.GetParams(ctx).GetAdminPolicyAccount(types.Policy_Type_add_observer) {
 		return &types.MsgAddObserverResponse{}, types.ErrNotAuthorizedPolicy
 	}
+	pubkey, _ := common.NewPubKey(msg.ZetaclientGranteePubkey)
+	granteeAddress, _ := common.GetAddressFromPubkeyString(msg.ZetaclientGranteePubkey)
+	k.DisableInboundOnly(ctx)
+	// AddNodeAccountOnly flag usage
+	// True: adds observer into the Node Account list but returns without adding to the observer list
+	// False: adds observer to the observer list, and not the node account list
```

Comment:
The `only` naming is a bit confusing to me because it would imply it's an action that is always executed but only this action is executed if the flag is set.